### ISSUE TITLE
enhancement (documentation, dashboard) AV1 encoder should be explained when it works

### ISSUE THREAD

**Issue by Meister1593** [alvr-org/ALVR#2331]
Seems like people mistakenly trying to use av1 even without either capable gpu, or headset, and get frustrated, or worse - thinking it could be even unimplemented feature, a bug, or something else

User should be told in case they don't have at least capable gpu, and if possible - headset too (quest 3)

**Comment by Maciejka1** [comment]
Yeah this would have saved me an hour today! In lastest nightly build it says that 7xxx or 30xx gpus are needed but no info about quest 3. 

![Image](https://github.com/user-attachments/assets/652f0c86-c393-4816-b5d9-ba2ffc5de838)

**Comment by a2cy** [comment]
The Nvidia RTX 30 series does not support AV1 encoding.
Only Nvidia RTX 40+ series supports AV1.

**Comment by akatsukinoyami** [comment]
Still have RTX 30xx in list of supported, that is not correct.
